.changeset/purple-cherries-fix.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-sentry': major
+---
+
+Added the possibility to specify organization per component
+
+**BREAKING**: Now the annotation `sentry.io/project-slug` can have the format of `[organization]/[project-slug]` or just `[project-slug]`.
```

Comment:
That change is actually not breaking, but tbh I think it's a better description so perhaps add it to the line above?

The breaking change is that change to the `fetchIssues` method signature

Reply:
Done!